library/chachapoly.c

```diff
@@ -368,9 +358,12 @@ int mbedtls_chachapoly_auth_decrypt( mbedtls_chachapoly_context *ctx,
     unsigned char check_tag[16];
     size_t i;
     int diff;
```

Comment:
I personally prefer a clear blank line between variable declarations and the first command.